proto/sifnode/ethbridge/v1/tx.proto

```diff
@@ -134,10 +134,10 @@ message MsgSetFeeInfo {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"minimum_burn_cost\""
   ];
-  string first_lock_double_peggy_cost = 7 [
+  string first_burn_double_peggy_cost = 7 [
```

Comment:
This is a bit strange, since we want to charge the fee on any export to an ethereum chain.  That could be either a lock or a burn.

Reply:
This fee just charged for creating a new contract on evm based blockchain. According to discussion before, we will use burn for "burning" imported token regradless the destination is back to original network or new network. So the cose just for burn case. Sifchain native token and ibc tokens, we need deploy contracts manually.